src/fundus/publishers/de/mdr.py

```diff
@@ -17,7 +18,15 @@
 class MDRParser(ParserProxy):
     class V1(BaseParser):
         _author_substitution_pattern: Pattern[str] = re.compile(r"MDR \w*$|MDR \w*-\w*$|MDRfragt-Redaktionsteam|^von")
-        _paragraph_selector = CSSSelector("div.paragraph")
+        _source_detection: str = (
+            r"((MDR AKTUELL \(ans\))|(Quell(e|en): [A-z\.]{3,4})|(\([A-z]{1,4}\))|([A-z]{3}/[A-z]{3}))"
+        )
```

Comment:
Could you provide an example for this? Best would be actually some regex101 example to show how the regex is supposed to work. IMO reading regexes it's always really hard to understand what's going on without having an example.

Reply:
I have now updated the regex and added a regex101 link

Reply:
Thanks 👍 The regex does not match within the next to last example. Is this on purpose?

Reply:
Yes, that is intended. That case was accidentally filtered out in a previous version of the regex.